openzeppelin/account/interfaces/IAccount.cairo

```diff
@@ -1,3 +1,6 @@
+# SPDX-License-Identifier: MIT
+# OpenZeppelin Cairo Contracts v0.1.0 (account/interfaces/IAccount.cairo)
```

Comment:
I don't think we need an interface directory when there's only one interface in there, the I prefix should be enough for most cases.